contracts/modules/claims/Claims.sol

```diff
@@ -262,40 +259,12 @@ contract Claims is Iupgradable {
     }
   }
 
-  /**
-  * @dev Pause Voting of All Pending Claims when Emergency Pause Start.
-  */
-  function pauseAllPendingClaimsVoting() public onlyInternal {
-    uint firstIndex = cd.pendingClaimStart();
-    uint actualClaimLength = cd.actualClaimLength();
-    for (uint i = firstIndex; i < actualClaimLength; i++) {
-      if (checkVoteClosing(i) == 0) {
-        uint dateUpd = cd.getClaimDateUpd(i);
-        cd.setPendingClaimDetails(i, (dateUpd.add(cd.maxVotingTime())).sub(now), false);
-      }
-    }
+  function pauseAllPendingClaimsVoting() external pure {
+    revert("Deprecated");
```

Comment:
Do not revert here

Reply:
let's enable tests that test this code path